mkdocs.yml

```diff
@@ -68,18 +76,19 @@ markdown_extensions:
   - pymdownx.details
   - pymdownx.highlight:
       linenums: false
-  - attr_list
-  - pymdownx.emoji:
-      emoji_index: !!python/name:materialx.emoji.twemoji
-      emoji_generator: !!python/name:materialx.emoji.to_svg
-
+    lightgallery: {}
+    pymdownx.inlinehilite: {}
+    pymdownx.emoji:
+        emoji_index: !!python/name:materialx.emoji.twemoji
+        emoji_generator: !!python/name:materialx.emoji.to_svg
 
 plugins:
   - search
   - git-revision-date
-  # - bootstrap-tables
-  - section-index # Adds links to nodes - comment out when creating PDF
-  #  - htmlproofer # Uncomment to check links - but extends build time significantly
+  - search
+  section-index: {}
+  search: 
+    separator: '[\s\-,:!=\[\]()"`/]+|\.(?!\d)|&[lg]t;|(?!\b)(?=[A-Z][a-z])'
   - macros:
       include_yaml:
 #        - 'variables.yml' # Use in markdown as '{{ VAR }}'
```

Comment:
Change the plugins section to match the following

plugins:
  section-index: {}
  search: 
    separator: '[\s\-,:!=\[\]()"`/]+|\.(?!\d)|&[lg]t;|(?!\b)(?=[A-Z][a-z])'
  git-revision-date: {}
  - macros:
      include_yaml:
#        - 'variables.yml' # Use in markdown as '{{ VAR }}'